redux-logger/redux-logger.d.ts

```diff
@@ -8,13 +8,15 @@
 declare module 'redux-logger' {
 
   interface ReduxLoggerOptions {
+    actionTransformer?: (action: any) => any;
     collapsed?: boolean;
+    duration?: boolean;
     level?: string;
     logger?: any;
+    predicate?: (getState: Function, action: any) => boolean;
     timestamp?: boolean;
-    transformer?: (state:any)=>any;
-    predicate?: (getState:Function, action:any)=>boolean;
+    transformer?: (state:any) => any;
```

Comment:
Redux's actions [must be plain objects](https://github.com/rackt/redux/blob/master/src/createStore.js#L95-L100), but I'm not sure about state. Maybe @gaearon can help?